tests/test_github.py

```diff
@@ -82,3 +82,58 @@ def test_download_issues():
     )
 
     github.download_issues("webcompat", "web-bugs", "all")
+
+
+def test_download_issues_updated_since_timestamp():
```

Comment:
Nit: `def test_download_issues_updated_since_timestamp() -> None:` so the function gets type checked.